config/dev/spm/sku_sival.yml

```diff
@@ -5,7 +5,6 @@
 sku: "prod_a"
 slotId: 0
 numSessions: 3
-highSecSeedKeyName: "HighSecKdfSeed"
-lowSecSeedKeyName: "LowSecKdfSeed"
-caKeyName: "KCAPriv"
-rootCAPath: certs/NuvotonTPMRootCA0200.cer
+symmetricKeys:
+  - name: "HighSecKdfSeed"
+  - name: "LowSecKdfSeed"
```

Comment:
I presume in a follow up you will load the Root CA, DICE ICA, and SKU-specific CA  keys for this sku too?

Reply:
Yes, I will do it in the same PR that enables loadtest for `EndorseCerts`.